beacon-chain/operations/service.go

```diff
@@ -112,18 +95,6 @@ func (s *Service) Status() error {
 	return nil
 }
 
-// IncomingExitFeed returns a feed that any service can send incoming p2p exits object into.
-// The beacon block operation pool service will subscribe to this feed in order to relay incoming exits.
-func (s *Service) IncomingExitFeed() *event.Feed {
```

Comment:
why is this outdated, dont we need to handle incoming validator exits ?

Reply:
Using `HandleValidatorExits` for that. There's no more feed